FAQ.md

```diff
@@ -16,13 +16,14 @@ Below are answers to common questions regarding OKD installation and administrat
 # Misc #
 - [Interesting commands while an upgrade runs](#interesting-commands-while-an-upgrade-runs)
 - [How can I find out what's inside of a (CI) release and which commit id each component has?](#how-can-i-find-out-whats-inside-of-a-ci-release-and-which-commit-id-each-component-has)
+- [How can I install OKD without libvirt libraries?](#how-can-i-install-okd-without-libvirt-libraries)
```

Comment:
this doesn't match the newly added heading

Reply:
Thank you very much for your review. I have corrected this.